data/campaigns/Eastern_Invasion/scenarios/04a_An_Elven_Interlude.cfg

```diff
@@ -278,11 +279,13 @@
         [/unit]
         [message]
             speaker=Nafga
+            #po: What are your orders for me to perform, Addogin?
```

Comment:
probably better was to change into: "my_order^And what about me?" as the same words could be used in different context

Reply:
but are they, though?

Reply:
While these ones aren't, the suggestion definitely does apply for separating S14's "No." when Owaec sees the Drowned Plains from the final battle's "No." when Dacyn refuses the bargain. It'll wait for 1.19, but I think those will become "despair^No." and "defiant refusal^No." respectively.